openeo_processes_dask/process_implementations/core.py

```diff
@@ -74,10 +74,11 @@ def wrapper(
             if arg not in inspect.signature(f).parameters:
                 resolved_kwargs.pop(arg, None)
 
-        pretty_args = {k: type(v) for k, v in resolved_kwargs.items()}
-        logger.warning(f"Running process {f.__name__}")
-        logger.warning(f"kwargs: {pretty_args}")
-        logger.warning("-" * 80)
+        pretty_args = {k: v.__repr__()[:80] for k, v in resolved_kwargs.items()}
```

Comment:
```suggestion
        pretty_args = {k: repr(v)[:80] for k, v in resolved_kwargs.items()}
```
would be a more canonical way of doing this